rnd/autogpt_builder/tailwind.config.ts

```diff
@@ -3,7 +3,9 @@ import type { Config } from "tailwindcss";
 const config = {
   darkMode: ["class"],
   content: [
-    './src/**/*.{ts,tsx}',
+    './components/**/*.{ts,tsx}',
+    './app/**/*.{ts,tsx}',
+    './src/**/*.{ts,tsx}'
```

Comment:
there's nothing outside of `src`

Reply:
Thank you for the call out on this I have reverted this change